.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -171,13 +171,34 @@ All actions should require re-authn with a webauthn device.
   - [ ] AWS OIDC card is rendered
   - [ ] AWS External Audit Storage is also rendered but locked behind a CTA
 
+- [ ] External Audit Storage 
```

Comment:
nit: there is a duplicate line above `AWS External Audit Storage renders only for enterprise cloud`?

Reply:
yeah that seems to be a test for whether the tile renders on OSS vs Enterprise cloud

these steps are what should be tested when going through the flow